sdk/lib/3rdparty/libxml2/trionan.c

```diff
@@ -327,7 +327,7 @@ trio_nan(TRIO_NOARGS)
 
   if (result == 0.0) {
 
-#if defined(TRIO_COMPILER_SUPPORTS_C99)
+#if defined(TRIO_COMPILER_SUPPORTS_C99) && !defined(__clang__)
```

Comment:
Either leave this of the one in `triodef.h`.

Reply:
IIRC I need both.

Reply:
What's the error here btw? I'd be surprised if clang-cl didn't have full C99 support.

Reply:
It was a quick fix to:
```
In file included from sdk\lib\3rdparty\libxml2\xpath.c:476:
sdk\lib\3rdparty\libxml2/trionan.c(332,14) :  warning: implicitly declaring library function 'nan' with type 'double (const char *)' [-Wimplicit-function-declaration]
    result = nan("");
             ^
```
Resulting in:
```
libxml2.lib(xpath.c.obj) : error LNK2019: unresolved external symbol _nan referenced in function _trio_nan
dll\3rdparty\libxslt\libxslt.dll : fatal error LNK1120: 1 unresolved externals
```

Reply:
I guess that's a deficiency in our CRT (which makes sense, Win2003's msvcrt doesn't support C99). So your fix seems fine.